### PR TITLE
Updates to the timeline in the FAQ so that it contains milestones up to today.

### PR DESCRIPTION
**Pull request by cybit**
The Timeline in the faq was lagging behind by a lot. This should be fixed now.

### CODE REVIEW

File: src/content/pages/faq.mdx [hunk 1]
```diff
@@ -23,11 +23,20 @@ subtitle: Frequently Asked Questions about EuroPython
...
-</ul>
-
-<ul className="milestone-todo">
 <li><b>2024-07-08</b>: Together at EuroPython 2024! 🎊 </li>
-</ul>
+  <li><b>2024-07-13</b>: Sprint Weekend (13-14 July)</li>
+  <li><b>2024-07-14</b>: End of EuroPython 2024</li>
+  <li><b>2024-09-13</b>:  Celebrate the [Team of Europython 2024 ](/thank-you)</li>
+  <li><b>2024-09-16</b>: [EuroPython 2024 Photos](/photos) Published</li>
+  <li><b>2024-08-03</b>: [EuroPython 2024 Videos](https://www.youtube.com/playlist?list=PL8uoeex94UhE1CbtkDK4hevp2lBif57Nq) Published</li>
```
Comment (hypha) [inline src/content/pages/faq.mdx:31]:
```suggestion
  <li><b>2024-10-03</b>: [EuroPython 2024 Videos](https://www.youtube.com/playlist?list=PL8uoeex94UhE1CbtkDK4hevp2lBif57Nq) Published</li>
```

File: src/content/pages/faq.mdx [hunk 2]
```diff
@@ -23,11 +23,20 @@ subtitle: Frequently Asked Questions about EuroPython
...
 <li><b>2024-06-14</b>: [Remote Ticket Sales](/tickets#remote-tickets) & [Financial Aid for Remote Participation](/finaid#-free-remote-ticket) start</li>
 <li><b>2024-06-30</b>: [Sprints Venue & Info](/sprints) announced</li>
-</ul>
-
-<ul className="milestone-todo">
 <li><b>2024-07-08</b>: Together at EuroPython 2024! 🎊 </li>
-</ul>
+  <li><b>2024-07-13</b>: Sprint Weekend (13-14 July)</li>
+  <li><b>2024-07-14</b>: End of EuroPython 2024</li>
+  <li><b>2024-09-13</b>:  Celebrate the [Team of Europython 2024 ](/thank-you)</li>
```
Comment (hypha) [inline src/content/pages/faq.mdx:29]:
```suggestion
  <li><b>2024-09-13</b>:  Celebrate the [Team of EuroPython 2024 ](/thank-you)</li>
```

File: src/content/pages/faq.mdx [hunk 3]
```diff
@@ -23,11 +23,20 @@ subtitle: Frequently Asked Questions about EuroPython
...
+  <li><b>2024-09-16</b>: [EuroPython 2024 Photos](/photos) Published</li>
+  <li><b>2024-08-03</b>: [EuroPython 2024 Videos](https://www.youtube.com/playlist?list=PL8uoeex94UhE1CbtkDK4hevp2lBif57Nq) Published</li>
+  </ul>
+
+[//]: # ()
+[//]: # (<ul className="milestone-todo">)
+
+[//]: # (<li><b>2024-12-01</b>: Let's get ready for EuroPython 2025! 🎊 </li>)
```
Comment (hypha) [inline src/content/pages/faq.mdx:37]: Aha is this deliberately commented out? Will this repo still be available for commits in Dec or might it have been archived and turned into static?

**Review by hypha — COMMENTED** [review]
Minor comments. LGTM otherwise.

File: src/content/pages/faq.mdx (hunk 1 above)
Comment (cybit) [reply src/content/pages/faq.mdx:31]: Outch. Thank you.

File: src/content/pages/faq.mdx (hunk 3 above)
Comment (cybit) [reply src/content/pages/faq.mdx:37]: Yes this is deliberate. I don't believe that anyone will ever uncomment this. But maybe this broad hint works. (You never know.)

**Review by hypha — APPROVED** [review]

**Pull request merged**